main/res/values/strings.xml

```diff
@@ -2029,27 +2029,27 @@
+    <string name="wizard_intro">Just a few steps are needed to configure c:geo and get you going. Shall I help you with that?</string>
+    <string name="wizard_intro2">Welcome back to the configuration wizard. Your c:geo setup is not yet finished, shall I help you to complete it and get you going? It\'s just a few steps.</string
     <string name="wizard_outro_ok">Thanks for configuring c:geo.\nHave fun!</string>
     <string name="wizard_outro_error">Incomplete configuration\nNot all parts of c:geo will work as expected!</string>
     <string name="wizard_permissions_title">Set app permissions</string>
-    <string name="wizard_permissions_intro">c:geo needs certain permissions to function correctly. I will explain them in the next steps.\nWithout these permissions you will not be able to use c:geo.</string>
-    <string name="wizard_platforms_title">Configure platforms</string>
-    <string name="wizard_platforms_intro">To download caches you need an account for at least one of the geocaching platforms supported by c:geo</string>
+    <string name="wizard_permissions_intro">c:geo needs certain permissions on your device to work correctly.\nI will explain more details in the next steps.</string>
+    <string name="wizard_platforms_title">Configure services</string>
+    <string name="wizard_platforms_intro">To find geocaches with c:geo you need an account for a geocaching service supported by c:geo.\nSelect the desired service below to either login with your existing account or create a new account.</string>
     <string name="wizard_platforms_gc">geocaching.com</string>
-    <string name="wizard_platforms_others">Opencaching &amp; more</string>
+    <string name="wizard_platforms_others">Opencaching &amp; Others</string>
     <string name="wizard_welcome_advanced">Advanced configuration</string>
     <string name="wizard_advanced_offlinemaps_label">Offline maps</string>
-    <string name="wizard_advanced_offlinemaps_info">c:geo can download map files for offline usage</string>
+    <string name="wizard_advanced_offlinemaps_info">This page provides some advanced functions, which are optional to use.\n\nc:geo can download map files for offline usage.</string>
     <string name="wizard_advanced_brouter_label">Routing app</string>
-    <string name="wizard_advanced_brouter_info">c:geo can navigate to targets and show routing info on your map by installing and configuring a companion app named \"BRouter Offline Navigation\"</string>
+    <string name="wizard_advanced_brouter_info">By installing and configuring a companion app named \"BRouter Offline Navigation\" c:geo can navigate to targets and show routing infos on your map.</string>
     <string name="wizard_advanced_restore_label">Restore</string>
-    <string name="wizard_advanced_restore_info">Restore settings / data from a previous installation</string>
+    <string name="wizard_advanced_restore_info">You can restore settings and/or geocache data from a previous installation.</string>
```

Comment:
"You can" may imply that there is stored data available. We should either check for existing backups (there's probably already a helper function for that) or adapt the text accordingly, e. g. "Try to restore ..."

Reply:
I will try to find a better wording not implying that a backup is available.

Reply:
Done